src/ReinforcementLearningCore/Project.toml

```
@@ -42,7 +43,7 @@ Parsers = "2"
 ProgressMeter = "1"
 Reexport = "1"
 ReinforcementLearningBase = "0.12"
-ReinforcementLearningTrajectories = "^0.3.2"
+ReinforcementLearningTrajectories = "0.3.3"
```

```suggestion
ReinforcementLearningTrajectories = "^0.3.3"
```
I think we want [0.3.3,0.4), right?